src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -284,6 +288,39 @@ class AppleDeviceInterface extends InterfacePrototype {
         })
     }
 
+    async addDeviceListeners () {
+        if (this.settings.featureToggles.third_party_credentials_provider) {
+            if (this.globalConfig.hasModernWebkitAPI) {
+                ddgGlobals.ObjectDefineProperty(ddgGlobals.window, 'providerStatusUpdated', {
+                    enumerable: false,
+                    configurable: false,
+                    writable: false,
+                    value: (data) => {
+                        this.providerStatusUpdated(data)
+                    }
+                })
+            } else {
+                // On Catalina we poll the native layer
+                setTimeout(() => this._pollForUpdatesToCredentialsProvider(), 2000)
```

Comment:
do we need this setTimeout? can catalina not just poll on the first call?

Reply:
We're already doing a lot on page startup, so I thought to keep the behaviour consistent and poll every 2s.